dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/airflow/AirflowConnector.java

```diff
@@ -0,0 +1,163 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.airflow;
+
+import com.google.auto.service.AutoService;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.edwmigration.dumper.application.dumper.ConnectorArguments;
+import com.google.edwmigration.dumper.application.dumper.annotations.RespectsArgumentDriverRequired;
+import com.google.edwmigration.dumper.application.dumper.annotations.RespectsArgumentPassword;
+import com.google.edwmigration.dumper.application.dumper.annotations.RespectsArgumentUser;
+import com.google.edwmigration.dumper.application.dumper.annotations.RespectsInput;
+import com.google.edwmigration.dumper.application.dumper.connector.AbstractJdbcConnector;
+import com.google.edwmigration.dumper.application.dumper.connector.Connector;
+import com.google.edwmigration.dumper.application.dumper.connector.MetadataConnector;
+import com.google.edwmigration.dumper.application.dumper.handle.Handle;
+import com.google.edwmigration.dumper.application.dumper.handle.JdbcHandle;
+import com.google.edwmigration.dumper.application.dumper.task.DumpMetadataTask;
+import com.google.edwmigration.dumper.application.dumper.task.FormatTask;
+import com.google.edwmigration.dumper.application.dumper.task.JdbcSelectTask;
+import com.google.edwmigration.dumper.application.dumper.task.Task;
+import com.google.edwmigration.dumper.application.dumper.task.TaskCategory;
+import com.google.edwmigration.dumper.application.dumper.utils.ArchiveNameUtil;
+import com.google.edwmigration.dumper.plugin.ext.jdk.annotation.Description;
+import java.sql.Driver;
+import java.time.Clock;
+import java.util.List;
+import javax.annotation.Nonnull;
+import javax.sql.DataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@AutoService(Connector.class)
+@Description("Dumps DAGs metadata from Airflow.")
+@RespectsInput(
+    order = 100,
+    arg = ConnectorArguments.OPT_HOST,
+    description = "Airflow database host.")
+@RespectsInput(
+    order = 200,
+    arg = ConnectorArguments.OPT_PORT,
+    description = "The port of the database.")
+@RespectsInput(
+    order = 250,
+    arg = ConnectorArguments.OPT_SCHEMA,
+    defaultValue = "airflow_db",
+    description = "Airflow database schema name")
+@RespectsArgumentDriverRequired
+@RespectsArgumentUser
+@RespectsArgumentPassword
+public class AirflowConnector extends AbstractJdbcConnector implements MetadataConnector {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AirflowConnector.class);
```

Comment:
Should not the LOG be in lower case?
https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

Reply:
yes, it should be. But I would like to keep it in upper case for consistency with other classes.

Reply:
Okay, it makes sense.